src/error.rs

```diff
@@ -35,16 +41,18 @@ pub enum BccError {
     InvalidTracepoint { message: String },
     #[error("userspace probe has invalid configuration: {message}")]
     InvalidUprobe { message: String },
+    #[error("USDT probe has invalid configuration: {message}")]
+    InvalidUSDT { message: String },
     #[error("error initializing perf map")]
     InitializePerfMap,
     #[error("error initializing ring buffer")]
     InitializeRingBuf,
     #[error("invalid cpu range ({range})")]
     InvalidCpuRange { range: String },
+    #[error("field '{field}' contained interior nul byte; can't convert to C string")]
```

Comment:
I'd probably use "null byte" - I don't see benefit in using a lowercase abbreviated form here.